src/test/sql/sql_translator_test.cpp

```diff
@@ -1875,6 +1875,15 @@ TEST_F(SQLTranslatorTest, ShowColumns) {
   EXPECT_LQP_EQ(actual_lqp, expected_lqp);
 }
 
+TEST_F(SQLTranslatorTest, SelectMetaTable) {
```

Comment:
We should have more complicated tests on the parts that call their own translator (e.g. subqueries?)